Cargo.toml

```diff
@@ -45,3 +45,5 @@ travis-ci = { repository = "tikv/grpc-rs" }
 
 [patch.crates-io]
 grpcio-compiler = { path = "compiler", version = "0.5.0", default-features = false }
+# Todo: This line should be removed after the new version of openssl-src is released (For now: 111.9.0+1.1.1g Apr 21, 2020)
+openssl-src = { git = "https://github.com/alexcrichton/openssl-src-rs.git", rev = "20bbf325bfa69b2c66614b3ab3f7f6e3239ca9ba", optional = true }
```

Comment:
Missing tailing empty line.

Reply:
Temporarily withdraw this commit to see if a new version of openssl-src will be released tomorrow